src/Common/constants.tsx

```diff
@@ -22,6 +26,17 @@ export interface OptionsType {
   disabled?: boolean;
 }
 
+export interface ConsultationTabProps {
+  // necessary props received for each tab
+  consultationData: ConsultationModel;
+  hl7SocketUrl?: string;
+  ventilatorSocketUrl?: string;
```

Comment:
That way you won't need to keep these in the typedef.

Reply:
removed